Lib/test/test_logging.py

```diff
@@ -3597,6 +3760,7 @@ def test_formatting(self):
                                             'deliberate mistake'))
         self.assertTrue(r.stack_info.startswith('Stack (most recent '
                                               'call last):\n'))
+        print(r.stack_info)
```

Comment:
You probably didn't mean to leave this line in? Was it there just for debugging?

Reply:
Yes! removing this now!